### ISSUE TITLE
Is there a particular method to use with Trigger Sequencer's run? 

### ISSUE THREAD

**Issue by spectromas** [countmodula/VCVRackPlugins#17]
Most modules with a run input can be hooked up to Impromptu Modular's Clocked run output and will start and stop according to the run status of Clocked. Trigger Sequencer doesn't seem to behave in the same way and if I re-open a patch then the sequencer will not run automatically like most others I've used, and pressing run on Clocked does not start it. 

I expect this is by design but what is the best method to make it start/stop along with Clocked?

**Comment by AdamVerspaget** [comment]
Hi, the run input essentially functions as an enable control. It requires a constant high gate signal to allow the sequencer to run. A low at the run input will stop the sequence. I'll run some checks over the weekend using clocked to find out what exactly is going on but I tested it with a manual gate module and it was functioning as I expected it to.

**Comment by AdamVerspaget** [comment]
OK, I have figured out what is happening. Clocked is outputting a really short trigger pulse each time you press the Run button rather than outputting a steady high gate level when the button is green and a steady low when it is not. Neither module is malfunctioning, they just operate differently. What you can do to get around this is use my Manual Gate module as a master to start and stop things. Send the "Trigger" out to the "Run" input on Clocked and the latched or "On" output to the trigger sequencer "Run" input and use the lovely big button to start and stop things. You may need to click the run button on Clocked once to get both in sync and may rest them both as well but after that they should play together nicely.

**Comment by spectromas** [comment]
Ok cool, thanks for the explanation and recommendation. I suspected it just worked differently rather than being a bug. 

I think I have found a simple workaround after some fiddling last night. The run on the sequencer isn't needed at all, at least for me currently, because just having one of Clocked's clocks (say that 10 times when you're drunk) plugged in to the Trigseq clock input, and Clocked's reset in to the trigseq's reset, the sequencer obviously stops when the clock stops and starts again accordingly (with a reset or not according to Clocked's right click settings). It does exactly what I was expecting and wanting. 

I'm fairly new to modular so it's always possible that I've misunderstood a function, perhaps that's the case here with 'run'.
